### PR TITLE
LPS-168816 Tests related to the Scope Card

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
This ticket contains the front-end unitary tests related to the Scope Card. 
[PR of the development](https://github.com/liferay-tango/liferay-portal/pull/3070)
[Link to the task](https://issues.liferay.com/browse/LPS-168816)
[Link to the story](https://issues.liferay.com/browse/LPS-167025)

Proposed Solution
========
After rendering the Scope Card component, it checks that the Scope title and the scope name are present.

### CODE REVIEW

File: modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 1]
```diff
@@ -180,6 +182,17 @@ const propertyGroups = [
 	},
 ];
 
+const scopeName = 'Liferay DXP';
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js:185]: If you only use this constant in the scope card test maybe you can move it there.

File: modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 2]
```diff
@@ -218,4 +231,26 @@ describe('ContributorsBuilder', () => {
...
+				propertyGroups={propertyGroups}
+				scopeName={scopeName}
+				supportedConjunctions={SUPPORTED_CONJUNCTIONS}
+				supportedOperators={SUPPORTED_OPERATORS}
+				supportedPropertyTypes={SUPPORTED_PROPERTY_TYPES}
+			/>
+		);
+
+		expect(getByText('scope')).toBeInTheDocument();
+		expect(getByText('Liferay DXP')).toBeInTheDocument();
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js:254]: You can reuse the scope variable here.

**Review by claraizquierdo — COMMENTED** [review]
Hi @Stephy6 , I have added some questions and comments.

File: modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 3]
```diff
@@ -180,6 +182,17 @@ const propertyGroups = [
 	},
 ];
 
+const scopeName = 'Liferay DXP';
+
+jest.mock('frontend-js-web', () => ({
```
[the remark this answers is not in the document]
Comment (Stephy6) [reply modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js]: I don't know, I thought it was necessary 😆

File: modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js (hunk 1 above)
Comment (Stephy6) [reply modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js]: Changed! thank u

File: modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js (hunk 2 above)
Comment (Stephy6) [reply modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js]: True! thank you!

**Review by claraizquierdo — APPROVED** [review]
LGTM!

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by Stephy6** [comment]
ci:test:relevant

**Comment by Stephy6** [comment]
ci:forward

**Pull request closed without merging**